web-client/src/presenter/actions/TrialSession/getBulkSpecialTrialSessionCopyNotesAction.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  SpecialTrialSession,
+  TrialSessionWorkingCopyNotes,
+} from '@shared/business/entities/trialSessions/SpeciailTrialSessions';
+const getSpecialTrialSessions = trialSessions =>
+  trialSessions
+    .filter(trialSession => trialSession.sessionType === 'Special')
```

Comment:
Let's do:
`import { SESSION_TYPES } from '@shared/business/entities/EntityConstants';` and then `SESSION_TYPES.special` instead of the raw string